dotnet.config

```diff
@@ -0,0 +1,2 @@
+[dotnet.test:runner]
+name= "MicrosoftTestingPlatform"
```

Comment:
Looks like the SDK version we currently have still doesn't have the rename that adds the dots. 😕

Reply:
```suggestion
[dotnet.test:runner]
name= "Microsoft.Testing.Platform"
```